cli/cmd/test_connection.go

```diff
@@ -318,16 +338,12 @@ func testPluginConnection(ctx context.Context, client plugin.PluginClient, spec
 
 // filterFailedTestResults fetch the failed test results.
 //
-// The function returns any failed test results, or nil if all tests passed. The hackernews plugin is excluded from the
-// failed test results since it was previously used as a test case for the test connection command.
+// The function returns any failed test results, or nil if all tests passed.
 func filterFailedTestResults(results []testConnectionResult) (*testConnectionResult, error) {
 	var failedResults []testConnectionResult
 
 	for _, result := range results {
 		if !result.Success {
-			if strings.Contains(result.PluginRef, "cloudquery/file@v4.0.4)") || strings.Contains(result.PluginRef, "cloudquery/hackernews@v3.0.25)") {
```

Comment:
This code is unnecessary because test connections have been using a relaxed spec reader for some time now, and dummy sources/dests are not included in cloud invocations anymore.